fivex/api/__init__.py

```diff
@@ -15,19 +15,19 @@
 
 
 @api_blueprint.route(
-    "/region/<string:chrom>/<int:start>-<int:end>/", methods=["GET"]
+    "/region/<string:chrom>/<int:start>-<int:end>/<string:study>/<string:tissue>/", methods=["GET"]
```

Comment:
In the previous design, tissue was optional (if omitted, data would be fetched from a merged dataset.

Does changing this to a url parameter represent a change in requirements? If so, I'll change the assoc track to warn if the field is ever absent:
https://github.com/statgen/fivex/blob/master/src/lz-helpers.js#L213-L224